ReSharper.FSharp/test/src/FSharp.Tests/QuickFixes/NamespaceToModuleTest.fs

```diff
@@ -0,0 +1,23 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Tests.Features
+
+open JetBrains.ReSharper.FeaturesTestFramework.Intentions
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Tests
+open NUnit.Framework
+
+[<FSharpTest>]
+type NamespaceToModuleTest() =
+    inherit QuickFixTestBase<NamespaceToModuleFix>()
+
+    override x.RelativeTestDataPath = "features/quickFixes/namespaceToModule"
+
+    [<Test>] member x.``Simple``() = x.DoNamedTest()
+
+
+[<FSharpTest>]
+type NamespaceToModuleAvailabilityTest() =
```

Comment:
Could you try if this attribute would work without creating the separate test class?

https://github.com/JetBrains/fsharp-support/blob/a52d76745e8d5beef7fb141b970f9168e40b7cb4/ReSharper.FSharp/test/src/FSharp.Tests/QuickFixes/ErrorAttributes.fs#L13-L14